composer/trainer/trainer.py

```diff
@@ -1320,12 +1321,44 @@ def __init__(
                     'Multiple concurrent uploads is not currently supported when using autoresume. Please set `num_concurrent_uploads` to 1 '
                     'for all `RemoteUploaderDownloader` instances.')
             assert latest_remote_file_name is not None
-            autoresume_checkpoint_path = self._get_autoresume_checkpoint(
-                save_folder=save_folder,
-                save_latest_filename=save_latest_filename,
-                save_latest_remote_file_name=latest_remote_file_name,
-                loggers=loggers,
-                load_progress_bar=load_progress_bar)
+            if self.state.fsdp_elastic_sharded_enabled:
+                ar_object_store = maybe_create_object_store_from_uri(save_folder)
+                # Symlink is on object store.
+                if ar_object_store is not None:
```

Comment:
doesn't this have to do a loop over loggers and find all the RUDs too? or are we just saying no support to autoresume + torch2 sharded + from your own RUD

Reply:
I mean this covers downloading from the save_folder right?

Reply:
yeah, but autoresume previously would loop over all loggers. Are we just saying that doesn't work for elastic sharding?

Reply:
correct